specification/components/schemas/FetchServicesList.yaml

```diff
@@ -0,0 +1,193 @@
+          example: 'https://fhir.nhs.uk/STU3/StructureDefinition/eRS-FetchServices-List-1'
+  resourceType:
+    type: string
+    enum:
+      - List
+    example: 'List'
+  status:
+    type: string
+    enum:
+      - current
+    example: 'current'
+  mode:
+    type: string
+    enum:
+      - snapshot
+    example: 'snapshot'
+  emptyReason:
+    type: object
+    required:
+      - coding
+    properties:
+      coding:
+        type: array
+        minItems: 1
+        maxItems: 1
+        items:
+          type: object
+          required:
+            - system
+            - code
+            - display
+          properties:
+            system:
+              type: string
+              enum:
+                - http://hl7.org/fhir/ValueSet/list-empty-reason
+              example: 'http://hl7.org/fhir/ValueSet/list-empty-reason'
+            code:
+              type: string
+              enum:
+                - nilknown
+                - notasked
+                - withheld
+                - unavailable
+                - notstarted
+                - closed
+              example: 'nilknown'
+            display:
+              type: string
+              example: 'no results'
+  entry:
+    type: array
+    items:
+      title: Singular entry with service details
+      type: object
+      required:
+       - item
+       - extension
+      properties:
+        extension:
+          type: array
+          items:
+            type: object
+            required:
+              - extension
+              - url
+            properties:
+              extension:
+                type: array
+                items:
+                  anyOf:
+                    - title: Extension to supply display prominently indicator
+                      type: object
+                      required:
+                        - url
+                        - valueBoolean
+                      properties:
+                        url:
+                          type: string
+                          enum:
+                            - displayProminently
+                          example: 'displayProminently'
+                        valueBoolean:
+                          type: boolean
+                          example: false
+                    - title: Extension to supply distance value
+                      type: object
+                      required:
+                        - url
+                        - valueUnsignedInt
+                      properties:
+                        url:
+                          type: string
+                          enum:
+                            - distance
+                          example: 'distance'
+                        valueUnsignedInt:
+                          type: integer
+                          format: int32
+                          example: 10
+                    - title: Extension to supply indicative wait time
```

Comment:
Is it worth adding some explanation to the title elements of these two properties to highlight that only one can ever be returned at once?